tgui/packages/tgui/components/BodyZoneSelector.tsx

```diff
@@ -74,7 +75,7 @@ export class BodyZoneSelector extends Component<
 
   render() {
     const { hoverZone } = this.state;
-    const { scale = 3, selectedZone } = this.props;
+    const { scale = 3, selectedZone, theme = 'midnight' } = this.props;
```

Comment:
If we're defaulting this, then change the type to `theme?: string` and remove the theme parameter from SurgeryInitiator

Reply:
done